### DISCUSSION TITLE
[FIRMWARE HELP] Disable buttons pressed color (RGB)

### DISCUSSION THREAD

**Discussion by Garouuuu** [OpenStickCommunity/GP2040-CE#639] [Q&A]
I want to disable button's pressed color , please help me , thanks ！

**Comment by TheTrainGoes** [comment]
Hi @Garouuuu ,

Please check out https://gp2040-ce.info/rgb-leds

Select + Start + P4 | Pressed Parameter Up
Select + Start + K4 | Pressed Parameter Down

Do this key combo to cycle through pressed colors until you reach no pressed color
